presto-product-tests/conf/tempto/tempto-configuration-for-docker-kerberos-simba.yaml

```diff
@@ -0,0 +1,47 @@
+hdfs:
```

Comment:
that's hell of a features in one config! :) Does it work without kerberos? (with the non-kerberized docker image?) If not, I insist we have that variant of the config too.